### PR TITLE
Added the OTel Profiling WG

### PR DESCRIPTION
**Pull request by iogbole**

### CODE REVIEW

File: README.md
Comment (arminru) [inline README.md:163]:
```suggestion
OpenTelemetry Profilng WG | Every other week on Thursday at 08:00 AM PT |[Google Doc](https://docs.google.com/document/d/1D-1MbT3MWpKM61JgVFx_TQi4vqWLLIZp6biK5MoHafI/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C03J794L0BV)|
```

**Review by arminru — APPROVED** [review]

File: README.md
Comment (trask) [inline README.md:163]: @Rperry2174 can you transfer this doc to the OpenTelemetry Governance google account?

from #1147:

> ## Process to transfer ownership
> 
> 1. Click the share button 
> 2. Invite cncf-opentelemetry-governance at lists.cncf.io to be an editor
> 3. Click the share button again
> 4. Click on the arrow next to the governance committee email and select "Transfer Ownership" 
> 5. Click "Send Invitation"

Comment (Rperry2174) [reply README.md:163]: this look correct? 
image

**Review by Rperry2174 — APPROVED** [review]
lgtm!

File: README.md
Comment (trask) [reply README.md:163]: yup!

**Comment by iogbole** [comment]
@trask - this PR requires one more approval from you, it seems.

File: README.md
Comment (Rperry2174) [reply README.md:163]: done!

**Comment by trask** [comment]
@iogbole @Rperry2174 it looks like you are still using the non-OpenTelemetry owned meeting notes, can you copy the latest changes from today's meeting over to the OpenTelemetry owned meeting notes and mark the old meeting notes as deprecated (and update the calendar event)? thx

**Comment by Rperry2174** [comment]
image

**Review by trask — APPROVED** [review]
thx!

**Comment by trask** [comment]
Closing and re-opening b/c github seems stuck on "Required statuses must pass before merging"

**Pull request merged**
